src/managed/OpenLiveWriter.CoreServices/HTML/HTMLDocumentHelper.cs

```diff
@@ -101,7 +101,7 @@ public static string EscapePath(string html, string url, string newUrl)
 #endif
 
             // When an absolute path doesn't end with a file or a slash (i.e. http://www.realultimatepower.net)
-            // converting it to a Uri automatically adds the trailing slash (i.e. http://www.realtultimatepower.net).
+            // converting it to a Uri automatically adds the trailing slash (i.e. http://www.realultimatepower.net).
```

Comment:
I've changed it, but, please note that err, the comment didn't actually make sense, so I've also added the trailing slash.

I can un-add the slash if you like, but..